drizzlepac/alignimages.py

```diff
@@ -486,11 +504,52 @@ def run_align(input_list, archive=False, clobber=False, debug=False, update_hdr_
         result.add_column(filteredTable[col], name=col)
     filteredTable.pprint(max_width=-1)
 
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+def match_relative_fit(imglist, reference_catalog,group_id_dict):
+    """Perform cross-matching and final fit using 2dHistogram matching
+
+    Parameters
+    ----------
+    imglist : list
+        List of input image `~tweakwcs.tpwcs.FITSWCS` objects with metadata and source catalogs
+
+    reference_catalog : Table
+        Astropy Table of reference sources for this field
+
```

Comment:
Add description for group_id_dict.

Reply:
Issue resolved by commit [619753a](https://github.com/spacetelescope/drizzlepac/pull/268/commits/619753a79c655136b4f7b3b2b452ace35abbf3af)